utide/__init__.py

```diff
@@ -20,8 +17,12 @@
 constit_index_dict = dict([(name, i) for (i, name) in
                            enumerate(constit_names)])
 
-from ._solve import solve  # noqa
-from ._reconstruct import reconstruct  # noqa
+from ._solve import solve
+from ._reconstruct import reconstruct
```

Comment:
These failed PEP8 before, but now they are good?

Reply:
Yep. flake8 was updated to understand those are used below in the `__all__` variable.

Reply:
Another quick question then, can we remove the allowed failures in Travis now?

Reply:
I'll work on that, there are some real pep8 infringements.
Note that, although I do like to follow code standards, I usually don't let those be real failures and drive the attention away from the code in PRs, so I recommend to leave them in the allowed failures.

Reply:
Yeah, I can agree on that, but I was more wondering why 3.6 passes and only 3.5 doesn't

Reply:
We only run `coding_standards` on Python 3.5, no need to run that more than once.